exercises/calculator.cpp

```diff
@@ -10,3 +10,22 @@
   Multiplication: 8
   Division: 2
 */
+
+
+#include <iostream>
+
+using namespace std;
+
+int main(){
+    int x,y;
+    cout<<"inserisci il primo valore"<<endl;
+    cin>>x;
+    cout<<"inserisci il seocndo valore"<<endl;
+    cin>>y;
+
+    cout<<"Somma: "<<x+y<<endl;
+    cout<<"Differenza: "<<x-y<<endl;
+    cout<<"Divisione: "<<x/y<<endl;
+    cout<<"Prodotto: "<<x*y<<endl;
+
```

Comment:
manca return 0
Inoltre, potresti spostare l'implementazione dell'esercizio in una funzione separata da main